posts/inside-rust/2024-08-12-announcing-metrics-initiative.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Announcing the Metrics Initiative"
+author: Jane Losare-Lusby and Esteban Kuber
+team: The Rust Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+We ([Jane](https://github.com/yaahc/) and [Esteban](https://github.com/estebank)) are excited to announce the start of the Metrics initiative.
```

Comment:
It woud be great if this explained what *is* the metrics initiative, and at the same time it also made it 100% clear that it is opt-in. This really needs to be the first thing presented in the blog post, or we'll have another flame war on our hands.

E.g. something like this:
```suggestion
We ([Jane](https://github.com/yaahc/) and [Esteban](https://github.com/estebank)) are excited to announce the start of the Metrics initiative, an **opt-in** and privacy-preserving system for recording various metrics and metadata from Rust compiler invocations. These metrics can then be optionally uploaded by Rust users to provide us with useful information about how Rust is being used in the wild.
```

Reply:
Just a reminder what would most likely happen if some people thought that this is on by default :laughing:
- https://www.reddit.com/r/golang/comments/10z5ig1/googles_go_may_add_telemetry_reporting_thats_on/
- https://www.theregister.com/2023/02/10/googles_go_programming_language_telemetry_debate/